docs/reference/search/request/stored-fields.asciidoc

```diff
@@ -40,7 +40,7 @@ GET /_search
 
 If the requested fields are not stored (`store` mapping set to `false`), they will be ignored.
 
-Field values fetched from the document it self are always returned as an array. Metadata fields like `_routing` and
+Field values fetched from the document itself are always returned as an array. Whereas metadata fields like `_routing` and
```

Comment:
I agree with the "Stored field values". But don't you think a conjunctive would make the contrast clearer? My intention was also to bolden the contrast. If you don't lie whereas, maybe something like "on the contrary"?